### PR TITLE
DCOS_OSS-938: Reduce steps to install catalog package.

### PR DESCRIPTION
**Pull request by MatApple**
Reduces click to install count.

[See visual](https://cl.ly/3f1R09323P2x/Screen%20Recording%202017-05-08%20at%2007.13%20PM.gif)

Closes [DCOS_OSS-938](https://jira.mesosphere.com/browse/DCOS_OSS-938)

**Need to write integration and system tests.**

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [x] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

cc/ @leemunroe

### CODE REVIEW

File: src/js/components/modals/__tests__/InstallPackageModal-test.js [hunk 1]
```diff
@@ -1,130 +0,0 @@
-jest.dontMock("../InstallPackageModal");
```
Comment (MatApple) [inline src/js/components/modals/__tests__/InstallPackageModal-test.js]: Unit tests are obsolete and should be replaced with integration/system tests which will come in a follow up commit.

File: src/js/components/modals/InstallPackageModal.js [hunk 2]
```diff
@@ -520,60 +496,9 @@ class InstallPackageModal
     );
   }
 
-  getCLIPackageInfo(cosmosPackage) {
```
Comment (MatApple) [inline src/js/components/modals/InstallPackageModal.js]: Modal will never be opened and have this method called.

File: src/js/components/modals/InstallPackageModal.js [hunk 3]
```diff
@@ -468,13 +445,12 @@ class InstallPackageModal
         </div>
         <div className="modal-footer">
           <div className="button-collection button-collection-stacked horizontal-center">
-            <button
-              disabled={!cosmosPackage || pendingRequest}
+            <a
               className="button button-success button-block"
-              onClick={this.props.onClose}
+              href={`#/services/detail/${encodeURIComponent(appId)}`}
```
Comment (MatApple) [inline src/js/components/modals/InstallPackageModal.js]: As discussed with @leemunroe.

**Comment by jfurrow** [comment]
I pushed one commit to remove the margin on the bottom of the error message, otherwise this looks great!

**Comment by weblancaster** [comment]
retest this please

**Comment by MatApple** [comment]
Thanks @weblancaster! Let's sync on the i18n stuff tomorrow 👍

**Comment by MatApple** [comment]
System tests will come in another PR.

**Pull request merged**